docs/resources/s3/object.md

```diff
@@ -15,8 +15,13 @@ Manages **S3 Objects** on IonosCloud.
 
```

Comment:
crossplane upjet parser also expects these files to be named as the resource but without prefix(`ionoscloud`).